packages/get-packages/README.md

```diff
@@ -1,21 +1,35 @@
 # @manypkg/get-packages
 
-> A simple utility to get the packages from a monorepo, whether they're using Yarn, Lerna, pnpm or Rush
+> A simple utility to get the packages from a monorepo, whether they're using Yarn, Bolt, Lerna, pnpm or Rush
 
-This library exports `getPackages` and `getPackagesSync`. It is intended mostly for use of developers building tools that want to support different kinds of monorepos as an easy way to write tools without having to write tool-specific code. It supports Yarn, Lerna, pnpm, Rush and single-package repos(where the only package is the the same as the root package). This library uses `@manypkg/find-root` to search up from the directory that's passed to `getPackages` or `getPackagesSync` to find the project root.
+This library exports `getPackages` and `getPackagesSync`. It is intended mostly for use of developers building tools that want to support different kinds of monorepos as an easy way to write tools without having to write tool-specific code. It supports Yarn, Bolt, Lerna, pnpm, Rush and single-package repos(where the only package is the the same as the root package). This library uses `@manypkg/find-root` to search up from the directory that's passed to `getPackages` or `getPackagesSync` to find the project root.
 
 ```typescript
 import { getPackages, getPackagesSync } from "@manypkg/get-packages";
 
-const { tool, root, packages } = await getPackages(process.cwd());
-const { tool, root, packages } = getPackagesSync(process.cwd());
+const { tool, packages, rootPackage, rootDir } = await getPackages(process.cwd());
+const { tool, packages, rootPackage, rootDir } = getPackagesSync(process.cwd());
 
-type Package = { packageJson: PackageJSON; dir: string };
+// From @manypkg/tools
 
-type Packages = {
+interface Tool {
+    readonly type: string;
+    isMonorepoRoot(directory: string): Promise<boolean>;
+    isMonorepoRootSync(directory: string): boolean;
+    getPackages(directory: string): Promise<Packages>;
+    getPackagesSync(directory: string): Packages;
+}
+
+interface Package {
+    packageJson: PackageJSON;
+    dir: string;
+    relativeDir: string;
+}
```

Comment:
I added `Tool` and `Package` here as `Packages` (below) refers to them. Not sure if that's needed/wanted. Happy to remove.

Reply:
perhaps we should refer to them as `import('@manypkg/tools').Tool` etc? This way we wouldn't have to worry about syncing those definitions in READMEs and it would be clear from where they are coming from here